### ISSUE TITLE
Add sonarr_series_year env variable to Custom Scripts

### ISSUE THREAD

**Issue by leonard4546** [Sonarr/Sonarr#5439]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Radarr has env variable radarr_movie_year, is it possible to add similar for Year of TV Show release. 

### Describe the solution you'd like

.

### Describe alternatives you've considered

.

**Comment by leonard4546** [comment]
@markus101 hey, is this not available on Sonarr V3? 

I have this version 
Version
3.0.10.1567
Mono Version
6.8.0.105

and it's not working for me.

**Comment by markus101** [comment]
No, it'll be in v4. https://github.com/Sonarr/Sonarr/commit/8d70def08873aa905f3eb491489d74cebc09fb9c if the branch list doesn't show `main` it's not in v3.

**Comment by leonard4546** [comment]
> No, it'll be in v4. [8d70def](https://github.com/Sonarr/Sonarr/commit/8d70def08873aa905f3eb491489d74cebc09fb9c) if the branch list doesn't show `main` it's not in v3.

is it possible to add this in upcoming V3 too? or development stopped for V3?

**Comment by markus101** [comment]
Only critical issues will be fixed in v3, no additional features will be added.
